v2-apache/Dockerfile

```diff
@@ -132,6 +121,7 @@ RUN set -eux; \
     apt-get install -qq -y --no-install-recommends --no-install-suggests \
         ca-certificates \
         libcurl3-gnutls \
+        libfuzzy \
```

Comment:
```suggestion
        libfuzzy2 \
```

Reply:
Thx, yes, it was confusing that the package and dev package are named differently.